.github/workflows/R-CMD-check.yaml

```diff
@@ -0,0 +1,102 @@
+# NOTE: This workflow is overkill for most R packages
+# check-standard.yaml is likely a better choice
+# usethis::use_github_action("check-standard") will install it.
+#
+# For help debugging build failures open an issue on the RStudio community with the 'github-actions' tag.
+# https://community.rstudio.com/new-topic?category=Package%20development&tags=github-actions
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
now that `current_web_functionality` is the default branch, does that mean that here "master" refers to `current_web_functionality`, or does it actually refer to the `master` branch? and is that what is intended?

Reply:
Great catch! Thanks! 
As is, I think this would be watching only on pushes and PR with the base branch `master`. I could change it to `current_web_functionality` but I instead I'll wait until we merge #136 -- which should be soon. Then `master` will be the correct base branch.